dev_guide/builds.adoc

```diff
@@ -1761,6 +1765,130 @@ This example uses `*pullSecret*` in a Source build, but it is also applicable
 in Docker and Custom builds.
 ====
 
+[[build-run-policy]]
+== Build Run Policy
+
+The build run policy describes the order in which the builds created from the
+build configuration should run. This can be done by changing the value of the
+`*runPolicy*` field in the `*spec*` section of the `*Build*` specification.
+
+It is also possible to change the `*runPolicy*` value for existing build
+configurations. 
+
+  * Changing `*Parallel*` to `*Serial*` or `*SerialLatestOnly*` and triggering a
+    new build from this configuration will cause the new build to wait until all
+    parallel builds complete as the "serial" build can only run alone.
+  * Changing `*Serial*` to `*SerialLatestOnly*` and triggering a new build will
+    cause cancellation of all existing builds in queue, except the currently
+    running build. The new build will execute as next.
+
+[[build-serial-run-policy]]
+=== Serial Run Policy
+
+Setting the `*runPolicy*` field to `*Serial*` will cause all new builds created
+from the `*Build*` configuration to be run in "serial" order. That means there
+will be only one build running at a time and every new build will wait until
+the previous build completes. Using this policy will result in consistent and
+predictable build output. This is the default `*runPolicy*`.
+
+
+Triggering three builds from the `sample-build` configuration, using the
+`*Serial*` policy will result in:
+
+====
+----
+NAME             TYPE      FROM          STATUS    STARTED          DURATION
+sample-build-1   Source    Git@e79d887   Running   13 seconds ago   13s
+sample-build-2   Source    Git           New
+sample-build-3   Source    Git           New
+----
+====
+
+
+When the `sample-build-1` build completes, the `sample-build-2` build will run:
+
+====
+----
+NAME             TYPE      FROM          STATUS    STARTED          DURATION
+sample-build-1   Source    Git@e79d887   Completed 43 seconds ago   34s
+sample-build-2   Source    Git@1aa381b   Running   2 seconds ago    2s
+sample-build-3   Source    Git           New
+----
+====
+
+
+[[build-serial-latest-only-run-policy]]
+=== SerialLatestOnly Run Policy
+
+Setting the `*runPolicy*` field to `*SerialLatestOnly*` will cause all new
+builds created from the `*Build*` configuration to be run in "serial" order,
+same as using the `*Serial*` run policy. The difference is that when a currently
+running build completes, the next build that will run is the latest build
+created. In other words, you don't wait for the queued builds to run as they
+will be skipped. This policy can be used for fast, iterative development. 
```

Comment:
"will be skipped.  Skipped builds are marked as \`\*Cancelled\*\`."  This policy can be..."